articles/azure-monitor/essentials/activity-log.md

```diff
@@ -96,6 +96,9 @@ AzureActivity
 | where CategoryValue == "Administrative"
 ```
 
+> [!Important]
+> In some scenarios, it is possible that values in fields of AzureActivity might have different casings from otherwise equivalent values. Take care when querying data in AzureActivity to utilize case insensitive operators for string comparisons, or use a scalar function to force a field to a uniform casing before any comparisons. For example, utilziing the [tolower()](/azure/kusto/query/tolowerfunction) function on a field to force it to always be lowercase, or the [=~ operator](/azure/kusto/query/datatypes-string-operators) when performing a string comparison.
```

Comment:
```suggestion
> [!Important]
> In some scenarios, it's possible that values in fields of AzureActivity might have different casings from otherwise equivalent values. Take care when querying data in AzureActivity to use case-insensitive operators for string comparisons, or use a scalar function to force a field to a uniform casing before any comparisons. For example, using the [tolower()](/azure/kusto/query/tolowerfunction) function on a field to force it to always be lowercase, or the [=~ operator](/azure/kusto/query/datatypes-string-operators) when performing a string comparison.
```